test/test_bundle.js

```diff
@@ -233,6 +233,194 @@ describe('json-pollock tests', function () {
     }
   };
 
+  var carouselSelectConf = {
+    type: "carouselSelect",
+    padding: 10,
+    selectMode: {
+      name: "test-carousel-multi-select",
+      type: "multiple", // "single" // selected items - which user has been selected
+    },
+    style: {
+      "border-color-selected": "red",
+    },
+    elements: [
+      {
+        type: "vertical",
+        meta: {
```

Comment:
@dvakulenko I think you missed updating the meta field to metadata here and a couple other places in the tests